<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MunkanapValt</class>
 <widget class="QDialog" name="MunkanapValt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>256</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mentButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Mentés</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Mégsem</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
